app/scripts/controllers/mmi-controller.test.ts

```diff
@@ -628,7 +657,7 @@ describe('MMIController', function () {
       mmiController.custodyController.getAccountDetails = jest
         .fn()
         .mockReturnValue({});
-      mmiController.messenger.call = jest
+      mmiController.messagingSystem.call = jest
```

Comment:
Hmm, this still isn't quite right. The `messagingSystem` should not be accessible outside of the controller.

I notice that this file is not being type-checked at all. It seems like someone at  some point added `// @ts-nocheck` at the top of the file. Therefore you're not getting any help whatsoever from TypeScript.

I guess fixing that is out of scope for this ticket but that is why TypeScript is not complaining here. :(

Reply:
Great, thank you!